front-end-admin/src/pages/user-interaction-management/components/userManagement/viewUserSection.tsx

```
@@ -0,0 +1,308 @@
+  // const {
+  //   data: userData,
+  //   isLoading: isFetchingUserData,
+  //   isError: isFetchingUserError,
+  // } = useDeactiveUserAndActive(useridparam, userStatus)
+  const { t } = useTranslation('pos')
+
+  const schema = yup
+    .object({
+      prefix: yup.string().when('_', {
+        is: () => 'tooltip',
+        then: yup
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)